tox.ini

```diff
@@ -14,6 +14,7 @@ basepython = python3
 ignore_errors = True
 skip_install = True
 deps =
+	distro
```

Comment:
That's redundant.

Reply:
If I remove it then the QA check fails:
```
qa: commands[0]> mypy ecleankernel test
ecleankernel/layout/std.py:5: error: Cannot find implementation or library stub for module named "distro"  [import-not-found]
ecleankernel/layout/blspec.py:5: error: Cannot find implementation or library stub for module named "distro"  [import-not-found]
ecleankernel/layout/blspec.py:5: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
test/test_layout_std.py:6: error: Cannot find implementation or library stub for module named "distro"  [import-not-found]
Found 3 errors in 3 files (checked 22 source files)
```

Reply:
That's a bug in `tox.ini`. Move the `mypy` call into the main testenv, like I do in https://github.com/projg2/pycargoebuild/blob/master/tox.ini

Reply:
Yes this works, Thanks. Fixed!